cli/tests/pcluster_config/test_pcluster_config_convert/test_slurm_unrelated_sections/pcluster.config.ini

```diff
@@ -62,13 +62,13 @@ fsx_fs_id = fs-0aaae053900f84047
 
 [ebs settings1]
 shared_dir = sharedebs
-volume_type = gp2
 volume_size = 50
 
 [ebs settings2]
 shared_dir = sharedebs2
 volume_type = gp2
```

Comment:
if we want to add iops here we need to change volume_type otherwise you get an invalid output:
```
[ebs settings2]
shared_dir = sharedebs2
volume_size = 10
volume_iops = 200
```

Reply:
Done, change it to gp3 volume